docs/design/nab_and_nar_status_update.md

```diff
@@ -124,69 +124,107 @@ It is similar for a NonAdminRestore.
 ```mermaid
 %%{init: {'theme':'neutral'}}%%
 graph
-event1{{predicate: Create event NAB}} == If event is accepted ==>
 
-reconcileStart1[/Reconcile start\] ==>
 
-phaseNew["`status.phase to **New**`"] -. Requeue: false, err .-> reconcileStart1
-phaseNew ==>
+predicateUpdateNabEvent{{predicate: Accepted **update** event NAB}};
+predicateCreateNabEvent{{predicate: Accepted **create** event NAB}};
+predicateUpdateVBEvent{{predicate: Accepted **update** event Velero Backup}};
 
-reconcileExit1[\Requeue: true, nil/]
-reconcileExit1 ==> question(is NonAdminBackup Spec valid?) == yes ==> reconcileStart2
-question == no ==> reconcileStartInvalid1
+requeueTrueNil[\"Requeue: true"/];
+requeueFalseNil[\"Requeue: false"/];
+requeueFalseErr[\"Requeue: false, error"/];
+requeueFalseTerminalErr[\"Requeue: false, Terminal error"/];
 
-reconcileStart2[/Reconcile start\] ==>
+reconcileStartAcceptedPredicate[/Reconcile start\];
 
-conditionsAcceptedTrue["`status.conditions[Accepted] to **True**`"] -. Requeue: false, err .- reconcileStart2
-conditionsAcceptedTrue ==>
+questionPhaseStatusSetToNew{"Is status.phase: **New** ?"};
+questionConditionAcceptedTrue{"Is status.conditions[Accepted]: **True** ?"};
+questionIsNabValid{"Is NonAdminBackup Spec valid ?"};
+questionPhaseStatusSetToBackingOff{"Is status.phase: **BackingOff**
+ and status.conditions[Accepted]: **False** ?"};
+questionPhaseStatusSetToCreated{"Is status.phase: **Created**
+ and status.conditions[BackupScheduled]: **True** ?"};
+questionStatusVeleroBackupUUID{"Is status.VeleroBackup.NameUUID existing and valid ?"};
+questionSuccess{"Success ?"};
+questionSuccessCreateVB{"Success ?"}
+questionSuccessWithTerminalError{"Success ?"};
+questionSuccessWithNoRequeue{"Success ?"};
+questionSuccessGetVB{"Error ?"};
```

Comment:
all questions are `success?` except this one. Better to be consistent?

Reply:
This is special one, where we can get 3 states in one decision block or split it by success and error. I wanted to have this in the 2 blocks to easy distinguish between those states as they result in different action.